File: eObrazovanjeeFront/src/app/model/addStudijskaGodina.ts


export class addStudijskaGodina implements addStudijskaGodinaInterface{
	
	public nacinFinansiranja: any; //enum
    public godinaStudija: number;
	public studentId: number;
	
	
		
	constructor(studijskaGodinatCfg:addStudijskaGodinaInterface)
	{	
		
		this.nacinFinansiranja = studijskaGodinatCfg.nacinFinansiranja;	
        this.godinaStudija = studijskaGodinatCfg.godinaStudija;
		this.studentId = studijskaGodinatCfg.studentId;
	}
}

interface addStudijskaGodinaInterface{
	
	nacinFinansiranja: string;
    godinaStudija: number;
  
	studentId: number;
  
}

File: eObrazovanjee/src/main/java/com/ftn/eObrazovanjee/dto/AddStudijskaGodina.java
package com.ftn.eObrazovanjee.dto;

import com.ftn.eObrazovanjee.model.NacinFinansiranja;

public class AddStudijskaGodina {

	private NacinFinansiranja nacinFinansiranja;
	private int godinaStudija;
	private Long studentId;
	
	public AddStudijskaGodina() {
		super();
	}

	public AddStudijskaGodina(NacinFinansiranja nacinFinansiranja, int godinaStudija, Long studentId) {
		super();
		this.nacinFinansiranja = nacinFinansiranja;
		this.godinaStudija = godinaStudija;
		this.studentId = studentId;
	}

	public NacinFinansiranja getNacinFinansiranja() {
		return nacinFinansiranja;
	}

	public void setNacinFinansiranja(NacinFinansiranja nacinFinansiranja) {
		this.nacinFinansiranja = nacinFinansiranja;
	}

	public int getGodinaStudija() {
		return godinaStudija;
	}

	public void setGodinaStudija(int godinaStudija) {
		this.godinaStudija = godinaStudija;
	}

	public Long getStudentId() {
		return studentId;
	}

	public void setStudentId(Long studentId) {
		this.studentId = studentId;
	}
	
	
	
}
